v2.0.0: customise the value calculation

tdsymonds opened the pull request:
Make the value calculation extendable, by combining the two existing tags and allowing a user to choose the calculation in the variable.

A user can extend this logic further without making huge changes, by adding their logic to the `calculateValue()` method.

tdsymonds committed:
Soteria tag v2.0.0

Make the value calculation extendable, by combining the two existing tags and allowing a user to choose the calculation in the variable.

A user can extend this logic further without making huge changes, by adding their logic to the  method.

Change-Id: I65d2f2ac247b93c332a2f73818e623321ad82554

tdsymonds committed:
Update links at the top of README

Change-Id: Ia464a3692f9bdf0904a3da81c7c6e07cedaa45ac

tdsymonds committed:
Add links to the places in the code to change.

For how to extend the value calculation.

Change-Id: I1a0772351f708e64c83036cf1027a41f7137dc14

martinacocco commented on docs/README.md:
````diff
@@ -307,25 +306,41 @@ This service account needs to have permission to access the Firestore data.
+  ```
+  conversion_value = (1 - return_rate) * profit * quantity
+  ```
+  If you select this field, you can optionally override the name of the return
````
Just a nit: 

Would this read better as "If you select this **option**, you can optionally override the name of the return rate field in Firestore." - or maybe I'm misunderstanding something

tdsymonds replied:
Fair point. I've updated this

tdsymonds committed:
Small wording tweak on README.

Change-Id: I366f9bf900435ddef84b784f89a2a6750ee0ad22

tdsymonds committed:
Resolve issue with spacing on images.

Change-Id: I25cc73a6a4bf16283ceb20ad0232fd443a33ed1c

tdsymonds committed:
Update images to use the Google Tag.

The GTM web container now uses a Google Tag in place of the GA4 tag to send data to the server.

This change updates the images to show the latest configuration.

Change-Id: Ib21ef65c9ad1bcbd455f1b752f8d963dd7021242
